Client/Frontend/Settings/AppSettingsOptions.swift

```diff
@@ -597,7 +597,11 @@ class LoginsSetting: Setting {
             success: {
                 self.settings?.navigateToLoginsList()
             },
-            cancel: nil,
+            cancel: {
+                if let selectedRow = self.settings?.tableView.indexPathForSelectedRow {
+                    self.settings!.tableView.deselectRowAtIndexPath(selectedRow, animated: true)
```

Comment:
Just for stylistic reasons? Functionally, it shouldn't be any different, because the block can only be executed if `self.settings` is non-nil, right?